test/test_optim.py

```diff
@@ -2018,6 +2019,7 @@ def test_multiplicative_lr(self):
         scheduler = MultiplicativeLR(self.opt, lr_lambda=[lambda x1: 0.9, lambda x2: 0.8])
         self._test(scheduler, targets, epochs)
 
+    @parametrize("T_mult", [1, 2, 4])
     def test_CosineAnnealingWarmRestarts_lr1(self):
         iters = 100
         eta_min = 1e-10
```

Comment:
Shouldn't iterate through `T_mults` anymore below not that the test is parametrized